packages/config-management/manage-generator/src/generatorInvocations.ts

```diff
@@ -0,0 +1,50 @@
+import { GeneratorInvocationSchema } from "@fern-api/workspace-configuration";
+
+export const JAVA_GENERATOR_INVOCATION: GeneratorInvocationSchema = {
+    name: "fernapi/fern-java",
+    version: "0.0.84",
+    generate: true,
+    config: {
+        packagePrefix: "com",
+        mode: "client_and_server",
+    },
+};
+
+export const TYPESCRIPT_GENERATOR_INVOCATION: GeneratorInvocationSchema = {
+    name: "fernapi/fern-typescript",
+    version: "0.0.155",
+    generate: true,
+    config: {
+        mode: "client_and_server",
+    },
+};
+
+export const POSTMAN_GENERATOR_INVOCATION: GeneratorInvocationSchema = {
+    name: "fernapi/fern-postman",
+    version: "0.0.21",
+    generate: {
+        enabled: true,
+        output: "./generated-postman.json",
+    },
+};
+
+export const OPENAPI_GENERATOR_INVOCATION: GeneratorInvocationSchema = {
+    name: "fernapi/fern-openapi",
+    version: "0.0.4",
+    generate: {
+        enabled: true,
+        output: "./generated-openapi.yml",
+    },
+    config: {
+        format: "yaml",
+    },
+};
+
+export function getGeneratorInvocationsByName(): Record<string, GeneratorInvocationSchema> {
```

Comment:
nbd but generally a function like this can just be an object:

```ts
export const GENERATOR_INVOCATIONS: Record<string, GeneratorInvocationSchema> = {
  [JAVA_GENERATOR_INVOCATION.name]: JAVA_GENERATOR_INVOCATION,
  ...
}

// no need to export these anymore, all consumers should use GENERATOR_INVOCATIONS
const JAVA_GENERATOR_INVOCATION: GeneratorInvocationSchema = {
...
```

Also, I would make the map from something stronger than `string` so the lookups are typesafe, e.g.
```ts
export enum Generator {
  Java,
  TypeScript,
  ...
}

export const GENERATOR_INVOCATIONS: Record<Generator, GeneratorInvocationSchema> = {
```

Reply:
ah thanks -- i couldnt figure out how to statically construct a record with dynamic keys

Reply:
the name still needs to be `"fernapi/fern-java"`

Reply:
You can do `Java = “name”` in the enum